test/parallel/test-dotenv-edge-cases.js

```diff
@@ -137,6 +137,25 @@ describe('.env supports edge cases', () => {
     assert.strictEqual(child.code, 0);
   });
 
+  it('should handle lines that come after lines with only spaces (and tabs)', async () => {
+    // Ref: https://github.com/nodejs/node/issues/56686
+    const code = `
+      process.loadEnvFile('./lines-with-only-spaces.env');
+      require('assert').strictEqual(process.env.EMPTY_LINE, 'value after an empty line');
```

Comment:
```suggestion
      assert.strictEqual(process.env.EMPTY_LINE, 'value after an empty line');
```

Reply:
I recommend fixing the other tests in this file on a different pull-request.

Reply:
I'm also happy to open a follow up PR to update the other tests 🙂👍